### PR TITLE
Workaround Ubuntu 17.04 tray bug

### PR DESCRIPTION
**Pull request by ConorIA**
This is an untested workaround for #574 and #575. I have used the console.log, and the property does seem to be set. (though in my case I was changing `KDE` to `Unity`).

If one of you has a working Zesty setup, please test. I've build a [`.deb` (x64)](https://cloud.conr.ca/s/RXfiBV3NX8HZSDf/download) to save you from building. If no one has a 17.04 install, I can ask the reporter take it for a spin.

If this does work, we should let the folks over at https://github.com/electron/electron/issues/9046 know.

### CODE REVIEW

File: electron/main.js [hunk 1]
```diff
@@ -89,6 +89,15 @@ if (process.platform !== 'darwin') {
 }
 
 ///////////////////////////////////////////////////////////////////////////////
+// Fix indicator icon on Unity
+// Source: https://bugs.launchpad.net/ubuntu/+bug/1559249
+///////////////////////////////////////////////////////////////////////////////
+const de = process.env.XDG_CURRENT_DESKTOP
+if (de.includes("Unity")) {
```
Comment (ConorIA) [inline electron/main.js]: There might be a better way to do this test. We need to change from `Unity:Unity7` to `Unity`.

File: electron/main.js [hunk 2]
```diff
@@ -89,6 +89,15 @@ if (process.platform !== 'darwin') {
 }
 
 ///////////////////////////////////////////////////////////////////////////////
+// Fix indicator icon on Unity
+// Source: https://bugs.launchpad.net/ubuntu/+bug/1559249
+///////////////////////////////////////////////////////////////////////////////
+const de = process.env.XDG_CURRENT_DESKTOP;
+if (de.includes('Unity')) {
```
Comment (ConorIA) [inline electron/main.js]: There might be a better way to do this test. We need to change from Unity:Unity7 to Unity.

**Comment by ConorIA** [comment]
Went in with Virtualbox.
Success!

screenshot_20170415_120820

File: electron/main.js (hunk 2 above)
Comment (lipis) [inline electron/main.js]: Just skip using the variable `de` and instead `process.env.XDG_CURRENT_DESKTOP.includes..`

**Comment by lipis** [comment]
Looks good to me.. @mythsunwind wdyt?

**Comment by lipis** [comment]
and @ffflorian :)

**Comment by mythsunwind** [comment]
To me too. Let's merge it before the new release.

**Comment by lipis** [comment]
🍻 🇪🇺 🍺 🇨🇦

**Pull request merged**